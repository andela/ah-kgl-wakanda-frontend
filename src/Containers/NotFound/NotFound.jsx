import React, { Component } from 'react';
import './NotFound.css';

class NotFound extends Component {
  state = {
    title: 'Page Not Found',
  };

  render() {
    const { title } = this.state;
    return (
      <React.Fragment>
        <h1>{title}</h1>
      </React.Fragment>
    );
  }
}

export default NotFound;
