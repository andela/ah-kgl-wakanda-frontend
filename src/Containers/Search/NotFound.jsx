import React from 'react';
import Image from 'react-bootstrap/Image';
import 'bootstrap/dist/css/bootstrap.css';
import imgNotFound from '../../assets/images/undraw_not_found.svg';

const imgStyle = {
  width: '45%',
};

/**
 * render articles not found in the search
 * @returns {object} jsx
 */
export default function NotFound() {
  return (
    <React.Fragment>
      <div className="mx-auto mt-5 py-5" style={imgStyle}>
        <Image src={imgNotFound} fluid alt="not_article_found" />
      </div>
      <h1 className="text-center">No result found for your search</h1>
    </React.Fragment>
  );
}
