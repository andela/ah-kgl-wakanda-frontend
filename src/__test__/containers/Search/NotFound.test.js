import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../../Containers/Search/NotFound';

describe('NotFound component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<NotFound />);
  });

  it('should render NotFound component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
