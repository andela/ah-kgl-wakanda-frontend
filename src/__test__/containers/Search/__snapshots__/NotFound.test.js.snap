// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should render NotFound component 1`] = `ShallowWrapper {}`;
